a3p-integration/proposals/a:vaults-auctions/agd-tools.js

```diff
@@ -0,0 +1,205 @@
+import {
+  agd,
+  agops,
+  agopsLocation,
+  CHAINID,
+  executeCommand,
+  executeOffer,
+  GOV1ADDR,
+  GOV2ADDR,
+  GOV3ADDR,
+  VALIDATORADDR,
+  waitForBlock,
+} from '@agoric/synthetic-chain';
+
+const ORACLE_ADDRESSES = [GOV1ADDR, GOV2ADDR, GOV3ADDR];
+
+export const BID_OFFER_ID = 'bid-vaultUpgrade-test3';
```

Comment:
this is specific to the current test and not a concern of using agd. please move out of this module.